SpiralCompiler/Syntax/Token.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpiralCompiler.Syntax
+{
+    public record struct Range(int Start, int End);
+
+    public enum TokenType
+    {
+        KeywordIf,      // if *
+        KeywordElse,    // else *
+        KeywordWhile,   // while *
+        KeywordFor,     // for *
+        KeywordIn,      // in *
+        Range,          // .. *
+        KeywordVar,     // var *
+        KeywordFunc,    // func *
+        KeywordClass,   // class *
+        KeywordThis,    // this *
+        KeywordReturn,  // return *
+        KeywordNew,     // new *
+        Dot,            // . *
+        Identifier,     
+        Integer,        // 123 *
+        Double,         // 1.23 *
+        String,         // "123" *
+        OpenBrace,      // { *
+        CloseBrace,     // } *
+        OpenParen,      // ( *
+        CloseParen,     // ) *
```

Comment:
If you had the convention to put the "category" first for keywords (KeywordA, KeywordB, ...)
then why not follow similarly for others? Like BraceOpen, BraceClose, ...